DEA_datasets/DEA_Coastlines.ipynb

```diff
@@ -0,0 +1,1225 @@
+{
```

Comment:
<p>Update the date :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/698/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>